### PR TITLE
Issue #24 Follow up

### PR DESCRIPTION
**Pull request by renehernandez**
* Add new functions in the module manifest
* Update usage doc to show how to change caller scope

### CODE REVIEW

**Comment by renehernandez** [comment]
@EsOsO Originally, I thought that the docs under the functions folder were automatically generated, but it seems that my assumption there is wrong, because there is no file for the new functions that I added in the [previous pull request](https://github.com/EsOsO/Logging/pull/25).

Should I update this pull request with the new functions docs? If so, do you have any guidelines on that area?

**Comment by EsOsO** [comment]
Hi @renehernandez, thanks for your contribution, are you sure that `Set-LoggingCallerScope` and `Get-LoggingCallerScope` needs to be exported in the module? Can we put in the private folder?

For the docs, you are right, the pages are generated only for exported functions; I'll fix the build script to update the `FunctionsToExport` automatically.

File: Logging/Logging.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 RootModule = 'Logging.psm1'
 
 # Version number of this module.
-ModuleVersion = '2.6.13'
+ModuleVersion = '2.6.15'
```
Comment (EsOsO) [inline Logging/Logging.psd1]: Please do not update `ModuleVersion` as it is updated by the build script using gitversion semver.

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-## 2.6.13 (2018-10-24)
+## 2.6.15 (2018-11-05)
```
Comment (EsOsO) [inline docs/CHANGELOG.md]: No need to add version, date ad infos in the changelog. Put your changes in `RELEASE.md` and the build script will insert it in the `CHANGELOG.md`

File: Logging/Logging.psd1 (hunk 1 above)
Comment (renehernandez) [reply Logging/Logging.psd1]: Oh I see what happens here. I pulled your master into my fork, which brought these updates to the manifest file. I didn't modify these by myself. Same with  the changelog file below

**Comment by renehernandez** [comment]
Hi @EsOsO, I actually need them for the use case that I added into the Readme. I have a wrapper function for writing the logs and the default caller scope will always print that function as the caller name if left unchanged

File: Logging/Logging.psd1 (hunk 1 above)
Comment (EsOsO) [reply Logging/Logging.psd1]: Ah ok, so no problem

**Comment by EsOsO** [comment]
I apologize, I'm facing issues with gitversion on appveyor bulding the module, I'll publish asap.

**Pull request merged**